ci-operator/config/openshift/ovs-cni/openshift-ovs-cni-main.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  ocp_4.19_base-rhel9:
+    name: "4.19"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-8-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.23-openshift-4.19
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.23-openshift-4.19
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.23
```

Comment:
Hi @zeeke,
This image doesn't exist it should also point to `rhel-9-release-golang-1.23-openshift-4.19`

Reply:
you're right. Updating